core/src/main/java/io/questdb/tasks/TelemetryTask.java

```diff
@@ -49,7 +49,7 @@ public QueryBuilder getCreateSql(QueryBuilder builder) {
                             "created timestamp, " +
                             "event short, " +
                             "origin short" +
-                            ") timestamp(created)"
+                            ") TIMESTAMP(created) PARTITION BY DAY TTL 1 WEEK BYPASS WAL"
```

Comment:
why to skip WAL?

Reply:
Originally it wasn't even partitioned, so I assumed that implied no WAL as well. Telemetry has a complete separate table that functions as a de-facto WAL, right?